frameworks/g-cloud-9/questions/services/webChatSupport.yml

```diff
@@ -22,10 +22,13 @@ type: radios
 options:
   - label: "Yes"
     value: "yes"
+    filter_label: "Web chat"
   - label: Yes, at an extra cost
     value: yes_extra_cost
+    filter_label: "Web chat (extra cost)"
```

Comment:
Yeah, this one will go away in favour of a new derived question, per <https://trello.com/c/BvnMwJZa/431-combining-web-chat-support-filter-options>. Just wanted to get something in there for demo purposes really.